contracts/Pool/PoolRegistry.sol

```diff
@@ -323,8 +326,8 @@ contract PoolRegistry is Ownable2StepUpgradeable, AccessControlledV8, PoolRegist
 
         IERC20Upgradeable token = IERC20Upgradeable(input.asset);
         uint256 amountToSupply = _transferIn(token, msg.sender, input.initialSupply);
-        token.approve(address(vToken), 0);
-        token.approve(address(vToken), amountToSupply);
+        token.safeApprove(address(vToken), 0);
+        token.safeApprove(address(vToken), amountToSupply);
```

Comment:
Is it intended to revert 'approve' to 'safeApprove' or did it happen by chance?

Reply:
Oups! Might be some weird rebase artifact. Good catch!

Reply:
Fixed in c13c409